ansible/modules/hashivault/hashivault_identity_group_alias.py

```diff
@@ -0,0 +1,220 @@
+description:
+    - Module to manage identity group aliases in Hashicorp Vault.
+options:
+    url:
+        description:
+            - url for vault
+        default: to environment variable VAULT_ADDR
+    ca_cert:
+        description:
+            - "path to a PEM-encoded CA cert file to use to verify the Vault server TLS certificate"
+        default: to environment variable VAULT_CACERT
+    ca_path:
+        description:
+            - "path to a directory of PEM-encoded CA cert files to verify the Vault server TLS certificate : if ca_cert
+             is specified, its value will take precedence"
+        default: to environment variable VAULT_CAPATH
+    client_cert:
+        description:
+            - "path to a PEM-encoded client certificate for TLS authentication to the Vault server"
+        default: to environment variable VAULT_CLIENT_CERT
+    client_key:
+        description:
+            - "path to an unencrypted PEM-encoded private key matching the client certificate"
+        default: to environment variable VAULT_CLIENT_KEY
+    verify:
+        description:
+            - "if set, do not verify presented TLS certificate before communicating with Vault server : setting this
+             variable is not recommended except during testing"
+        default: to environment variable VAULT_SKIP_VERIFY
+    authtype:
+        description:
+            - "authentication type to use: token, userpass, github, ldap, approle, oidc"
+        default: token
+    token:
+        description:
+            - token for vault
+        default: to environment variable VAULT_TOKEN
+    username:
+        description:
+            - username to login to vault.
+        default: to environment variable VAULT_USER
+    password:
+        description:
+            - password to login to vault.
+        default: to environment variable VAULT_PASSWORD
+    name:
+        description:
+            - Name of the alias. Name should be the identifier of the client in the authentication source.
+    alias_id:
+        description:
+            - ID of the group alias. If set, updates the corresponding group alias.
+    group_name:
+        description:
+            - Group name to which this alias belongs to.
+    canonical_id:
+        description:
+            - Group ID to which this alias belongs to.
+    mount_accessor:
+        description:
+            - Accessor of the mount to which the alias should belong to.
+    state:
+        description:
+            - whether create/update or delete the group
+'''
+EXAMPLES = '''
+---
+- hosts: localhost
+  tasks:
+    - hashivault_identity_group_alias:
+      name: 'admins'
+      group_name: 'admins'
+      authtype: 'ldap'
+'''
+
+
+def main():
+    argspec = hashivault_argspec()
+    argspec['name'] = dict(required=True, type='str', default=None)
+    argspec['alias_id'] = dict(required=False, type='str', default=None)
+    argspec['group_name'] = dict(required=False, type='str', defualt=None)
```

Comment:
change defualt to default (typo imported from hashivault_identity_entity_alias)

Reply:
done